main.rb

```diff
@@ -1,37 +1,30 @@
 require_relative 'app'
```

Comment:
- [ ] It would be nice if you made your app.rb file Single Responsible because it should only take one responsibility because right now it is taking multiple responsibilities of listing the data and getting the data from the user and manipulating the data so your app.rb file should be Single Respoibile 👍 You can make it Single Responsible by putting the input methods into those classes which are responsible for handling them 😊 

**NOTE: If you don't understand it then don't hesitate to reach out to me on Slack by searching my name Hammas in the full-time-students channel or you can also call me on zoom for more clarification if you need any 👍**